addons/mail/views/mail_activity_views.xml

```diff
@@ -194,6 +194,25 @@
         </field>
     </record>
 
+    <record id="mail_activity_view_form_without_record_access" model="ir.ui.view">
+        <field name="name">mail.activity.view.form.without.record.access</field>
+        <field name="model">mail.activity</field>
+        <field name="priority">32</field>
+        <field name="arch" type="xml">
+            <form string="Log an Activity" create="false">
```

Comment:
Note: I can delete the record from the form view, not sure if intended?

Reply:
No indeed, thanks. I have removed it as well.